pillar/elife.sls

```diff
@@ -183,6 +183,8 @@ elife:
                 #    command: /srv/annotations/bin/console queue:watch
                 #    number: 1
                 #    [require: some-state]
+    proofreader_php:
+        version: 0.1
```

Comment:
Since we'll mostly just be adding rules rather than changing them, we can update the code first then bump this, rather than having to declare the version in the repo.

Reply:
Interesting, let's test that out with medium.